backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/bpartner_pricelist/BpartnerPriceListServicesFacade.java

```diff
@@ -93,9 +101,10 @@ public PriceListVersionId getPriceListVersionId(final PriceListId priceListId, f
 		return priceListsRepo.retrievePriceListVersionId(priceListId, date);
 	}
 
-	public ImmutableList<I_M_ProductPrice> getProductPrices(PriceListVersionId priceListVersionId)
+	public ImmutableList<ProductPrice> getProductPrices(final PriceListVersionId priceListVersionId)
```

Comment:
please annotate parameters as  `@Nullable` or `@NonNull` whereever you touch the code and it's possible